diff --git a/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle b/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
index 4bb66ea..b9cc113 100644
--- a/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
+++ b/node_modules/@codetrix-studio/capacitor-google-auth/android/build.gradle
@@ -7,8 +7,8 @@ ext {
 
 buildscript {
     repositories {
-        jcenter()
         google()
+        mavenCentral()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:8.2.1'
@@ -31,7 +31,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
     lintOptions {
@@ -45,7 +45,6 @@ android {
 
 repositories {
     google()
-    jcenter()
     mavenCentral()
 }
 
diff --git a/node_modules/@codetrix-studio/capacitor-google-auth/android/src/main/java/com/codetrixstudio/capacitor/GoogleAuth/GoogleAuth.java b/node_modules/@codetrix-studio/capacitor-google-auth/android/src/main/java/com/codetrixstudio/capacitor/GoogleAuth/GoogleAuth.java
index 147fac8..dbc3a32 100644
--- a/node_modules/@codetrix-studio/capacitor-google-auth/android/src/main/java/com/codetrixstudio/capacitor/GoogleAuth/GoogleAuth.java
+++ b/node_modules/@codetrix-studio/capacitor-google-auth/android/src/main/java/com/codetrixstudio/capacitor/GoogleAuth/GoogleAuth.java
@@ -54,6 +54,7 @@ public class GoogleAuth extends Plugin {
   private GoogleSignInClient googleSignInClient;
 
   public void loadSignInClient (String clientId, boolean forceCodeForRefreshToken, String[] scopeArray) {
+    Log.d("GoogleAuthPlugin", "loadSignInClient called with clientId: " + clientId + ", forceCodeForRefreshToken: " + forceCodeForRefreshToken);
     GoogleSignInOptions.Builder googleSignInBuilder = new GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
       .requestIdToken(clientId)
       .requestEmail();
@@ -64,8 +65,10 @@ public class GoogleAuth extends Plugin {
 
     Scope[] scopes = new Scope[scopeArray.length - 1];
     Scope firstScope = new Scope(scopeArray[0]);
+    Log.d("GoogleAuthPlugin", "Scope 0: " + scopeArray[0]);
     for (int i = 1; i < scopeArray.length; i++) {
       scopes[i - 1] = new Scope(scopeArray[i]);
+      Log.d("GoogleAuthPlugin", "Scope " + i + ": " + scopeArray[i]);
     }
     googleSignInBuilder.requestScopes(firstScope, scopes);
 
@@ -193,6 +196,9 @@ public class GoogleAuth extends Plugin {
       // this is for scopes that are in the form of a url
       .replace("\\", "");
 
+    Log.d("GoogleAuthPlugin", "Raw scopesStr: " + scopesStr);
+    Log.d("GoogleAuthPlugin", "replacedScopesStr: " + replacedScopesStr);
+
     // scope to be in the form of an array
     String[] scopeArray = replacedScopesStr.split(",");
 
@@ -203,7 +209,7 @@ public class GoogleAuth extends Plugin {
   // Logic to retrieve accessToken, see https://github.com/EddyVerbruggen/cordova-plugin-googleplus/blob/master/src/android/GooglePlus.java
   private JSONObject getAuthToken(Account account, boolean retry) throws Exception {
     AccountManager manager = AccountManager.get(getContext());
-    AccountManagerFuture<Bundle> future = manager.getAuthToken(account, "oauth2:profile email", null, false, null, null);
+    AccountManagerFuture<Bundle> future = manager.getAuthToken(account, "oauth2:profile email https://www.googleapis.com/auth/calendar.events", null, false, null, null);
     Bundle bundle = future.getResult();
     String authToken = bundle.getString(AccountManager.KEY_AUTHTOKEN);
     try {
